envoy/api/v2/core/http_uri.proto

```diff
@@ -34,4 +38,7 @@ message HttpUri {
     //
     string cluster = 2 [(validate.rules).string.min_bytes = 1];
   }
+
+  // Sets the maximum duration in milliseconds that a response can take to arrive upon request.
```

Comment:
What's the default if not specified?

Reply:
Would be better to make this field required and >= 0? It's not clear to me when/how to enforce a default value.

Reply:
SGTM